packages/datadog-api-client-v1/.openapi-generator/FILES

```diff
@@ -515,6 +515,12 @@ models/TimeseriesWidgetRequest.ts
 models/ToplistWidgetDefinition.ts
 models/ToplistWidgetDefinitionType.ts
 models/ToplistWidgetRequest.ts
+models/TreeMapColorBy.ts
```

Comment:
suspicious too :)

Reply:
That's on me, I didn't notice the merge failed. Should be good now.